### ISSUE TITLE
Should `pnpm -g root` return `pnpm-global-pkg`?

### ISSUE THREAD

**Issue by vjpr** [pnpm/pnpm#850]
I am wondering whether it would be better for it to return the dir printed in `pnpm list -g`?

Depends on how `npm -g root` is most commonly used in practice. Do we rely on it? What does yarn do?

In nvm, [Migrating global packages while installing](https://github.com/creationix/nvm#migrating-global-packages-while-installing) may be broken currently - I'm not sure need to check what it uses...

The problem which lead me here was how to best access the dir where global modules are installed. `$(pnpm config get prefix)/pnpm-global/node_modules`? There is also `npm -g bin` -> `/Users/Vaughan/nvm/versions/node/v8.1.2/bin`

**Comment by zkochan** [comment]
Published with [v1.10.1](https://github.com/pnpm/pnpm/releases/tag/v1.10.1)

**Comment by andreineculau** [comment]
@zkochan @vjpr I'm not sure of the plain-english outcome of this? I am also running after "where are my global modules installed?" and currently pnpm and npm return different things:

```
$ npm root -g /usr/local/lib/node_modules

$ pnpm root -g
/usr/local/Cellar/node/8.4.0/pnpm-global/1/node_modules
```

the path that pnpm returns doesn't even exist actually
